### ISSUE TITLE
Show example response headers again

### ISSUE THREAD

**Issue by hanspagel** [scalar/scalar#302]
The response headers are hidden. We need to find a way to fix the height problem for long lists of headers before showing it again.

**Comment by ATREAY** [comment]
hey @hanspagel, I would like to work on this issue if it's open.
Additionally, could you provide a link or reference to the location of this issue?

**Comment by hanspagel** [comment]
This needs some thinking before we can do another attempt to add them! I’m not even sure what we want there, so I’d have a hard time approving any PR rn.

**Comment by ATREAY** [comment]
No worries, @hanspagel. If you ever decide to include this feature in the future, just give me a heads-up. I'd be happy to add it if it can be done.

**Comment by hanspagel** [comment]
Duplicate of #2939.
